Parsers/ASimAuthentication/ProductParsers/AuthenticationWindowsEvents.yaml

```diff
@@ -0,0 +1,136 @@
+                                              ActorSessionId = tostring(Data.SubjectLogonId),
+                                              ActorUsername = tostring(Data.SubjectUserName),
+                                              ActorUserId = tostring(Data.SubjectUserSid),
+                                              SubStatus = tostring(Data.SubStatus),
+                                              TargetDomainName = tostring(Data.TargetDomainName),
+                                              TargetSessionId = tostring(Data.TargetLogonId),
+                                              TargetUserId = tostring(Data.TargetUserSid),
+                                              TargetUsername = tostring(Data.TargetUserName),
+                                              SrcDvcHostname = tostring(Data.WorkstationName),
+                                              ActorUserAccount = tostring(Data.SubjectAccount),
+                                              TargetUserAccount = tostring(Data.TargetAccount),
+                                              EventProduct = "Microsoft-Windows-Security-Auditing"
+                                  | extend EventStatus= iff(SubStatus=='0x0',Status,SubStatus)
+                                  // -- creating EventMessage matching EventMessage in SecurityEvent table
+                                  | extend EventMessage = case(EventID == 4634, "4634 - An account was logged off.", 
+                                                              EventID == 4625, "4625 - An account failed to log on.",
+                                                              EventID == 4624 ,"4624 - An account was successfully logged on.",
+                                                              "4647 - User initiated logoff."),
+                                          EventResult = iff(EventID == 4625, 'Failure', 'Success')
+                                  | project-rename 
+                                      TargetDvcHostname = Computer
+                                  // , TargetUserType=AccountType - no AccountType in windowsEvents
+                                  // , EventOriginalUid = EventOriginId - no EventOriginalId in WindowsEvents
+                                  , EventOriginId=EventID
+                                  | extend  EventCount=int(1)
+                                          , EventSchemaVersion='1.0.0'
+                                          , ActorUserIdType='SID'
+                                          , TargetUserIdType='SID'
+                                          , EventVendor='Microsoft'  
+                                          , EventStartTime =TimeGenerated
+                                          , EventEndTime=TimeGenerated
+                                          , EventType=iff(EventOriginId in (LogoffEvents), 'Logoff', 'Logon')
+                                          , ActorUsername = tostring(Data.SubjectUserName)
```

Comment:
Need to be verified. The logic is based on SecurityEvent, which, as we learned, changes things.